blockchain/txvalidator.go

```diff
@@ -2160,6 +2160,42 @@ func (b *BlockChain) checkCRCProposalTransaction(txn *Transaction,
 		return errors.New("budgets exceeded the maximum limit")
 	}
 
+	if proposal.ProposalType == payload.ChangeProposalOwner {
+		if !b.crCommittee.IsInElectionPeriod() || b.crCommittee.IsInVotingPeriod(blockHeight) {
```

Comment:
change proposal owner transaction should be in election period and not in voting period . should it be like `b.crCommittee.IsInElectionPeriod() && !b.crCommittee.IsInVotingPeriod(blockHeight)`

Reply:
return error ?